CorsixTH/Lua/app.lua

```diff
@@ -118,7 +118,9 @@ function App:init()
   local good_install_folder, error_message = self:checkInstallFolder()
   self.good_install_folder = good_install_folder
   self.level_dir = debug.getinfo(1, "S").source:sub(2, -12) .. "Levels" .. pathsep
+  self.campaign_dir = debug.getinfo(1, "S").source:sub(2, -12) .. "Campaigns" .. pathsep
```

Comment:
Not your fault, but `debug.getinfo(1, "S").source:sub(2, -12)`?

What is debugging code doing here? Why isn't there a nice function-name for this madness?

Reply:
Unfortunately I can't answer. And I would assume as I can't I probably shouldn't try to guess?

Reply:
@tobylane answered in Discord. Toby I'm happy if you wanted to commit that change into this PR if you wanted?

Reply:
https://grep.app/search?q=Debug.getinfo&filter[repo][0]=CorsixTH/CorsixTH&filter[lang][0]=Lua&filter[path][0]=CorsixTH/

There's 19 uses of it. It can go here, but if you're ready before then it can go in a new PR.

Reply:
There's one in `CorsixTH.lua` -- not much you can do about that one in app, unless you set it in that file.

Reply:
If you're touching the entire codebase, I'd put it in a new PR